### PR TITLE
New Cheat Sheet: Libreoffice Impress

### PR DESCRIPTION
**Pull request by ambrish61**
## Type of Change

- [ X] New Instant Answer
    - [ X] Cheat Sheet
- [ ] Improvement
    - [ ] Bug fix
    - [ ] Enhancement
- [ ] Non–Instant Answer
    - [ ] Other (Role, Template, Test, Documentation, etc.)

## Description of new Instant Answer, or changes

This Cheat Sheet Provides Information about Keyboard Shortcuts of LibreOffice Impress. An Open Source Alternative of MS Office Powerpoint. 

## People to notify

---

IA Page: https://duck.co/ia/view/libre_office_impress_cheat_sheet

Forum Topic:

### CODE REVIEW

**Comment by daxtheduck** [comment]
@ambrish61 Thanks for the pull request! Unfortunately, we're only accepting contributions and improvements to a limited portion of the search space.
            
The staff and community are currently focused on making DuckDuckGo the [best search engine for programmers](https://forum.duckduckhack.com/t/duckduckhack-programming-mission-overview/53). To ensure we're making as much progress as possible, **we are only accepting Instant Answers and updates related to the programming mission**.
    
If this is not related to the Programming Mission, it will likely be closed but you can:
    
1. Help us prevent further confusion by telling us where you found out about DuckDuckHack or where we could've alerted you earlier.
2. Keep this code handy! We'll be expanding into other search topics in the future!
3. Join us at our new home: https://forum.duckduckhack.com/

If this is related to the Programming Mission, it will be reviewed by staff and the community as soon as possible!

**Comment by daxtheduck** [comment]
Metadata placeholder post
            
*If you haven't already, please edit your first comment to include a link to the IA page at the bottom.*
            
---
            
**How to find your IA page**
- New Instant Answer
  -- follow the steps [here](https://duck.co/ia/new_ia) to create your IA page.
              
- Existing Instant Answer
  -- find your IA page from this [list](https://duck.co/ia)

File: share/goodie/cheat_sheets/json/libre_office_impress.json [hunk 1]
```diff
@@ -0,0 +1,283 @@
...
+	"metadata": {
+             "sourceName": "LibreOffice Help",
+             "sourceUrl":  "https://help.libreoffice.org/Impress/Shortcut_Keys_for_Impress"
+	},
+	"aliases": [
+		     "libreoffice impress",
+		     "libre impress",
+		     "libreimpress"
+	],
+	"template_type": "Keyboard",
```
Comment (amarlearning) [inline share/goodie/cheat_sheets/json/libre_office_impress.json]: make it `keyboard` with a small 'k'

**Comment by amarlearning** [comment]
@ambrish61 change the file name to `libre-office-impress.json` instead of `libre_office_impress.json`

File: share/goodie/cheat_sheets/json/libre_office_impress.json [hunk 2]
```diff
@@ -0,0 +1,283 @@
...
+			"key":"[Ctrl] [Shift] [Space]",
+			"val":"Non-breaking spaces"
+		},{
+			"key":"←",
+			"val":"Move cursor to left"
+		},{
+			"key":"[Shift] [←]",
+			"val":"Move cursor with selection to the left"
+		},{
+            "key":"[Ctrl] [←]",
```
Comment (amarlearning) [inline share/goodie/cheat_sheets/json/libre_office_impress.json]: Not properly intended!

File: share/goodie/cheat_sheets/json/libre_office_impress.json [hunk 3]
```diff
@@ -0,0 +1,283 @@
...
+                "key": "[Shift] [drag]",
+                "val": "Constrains the movement of the selected object horizontally or vretically"
+        },  {
+                "key": "[Ctrl] [drag (with copy when moving active)]",
+                "val": "Hold down Ctrl and drag an object to create a copy of the object"
+        },  {
+                "key": "Alt",
+                "val": "Hold down Alt to draw or resize objects by dragging from the center of object outward"
+        },  {
+                "key": "[Alt] [click]  ",
```
Comment (amarlearning) [inline share/goodie/cheat_sheets/json/libre_office_impress.json]: remove that extra space from the end of `key`

File: share/goodie/cheat_sheets/json/libre_office_impress.json [hunk 4]
```diff
@@ -0,0 +1,283 @@
...
+                "key": "[Shift] [Tab]",
+                "val": "Select objects in the reverse order in which they were created "
+        },  {
+                "key": "Escape",
+                "val": "Exit current mode"
+        },  {
+                "key": "Enter",
+                "val": "Activate a placeholder object in a new presentation"
+        },  {
+                "key": "[Ctrl] [Enter] ",
```
Comment (amarlearning) [inline share/goodie/cheat_sheets/json/libre_office_impress.json]: Again an extra space!

Comment (ambrish61) [inline share/goodie/cheat_sheets/json/libre_office_impress.json]: Thanks @amarlearning for pointing out problems.

**Comment by jophab** [comment]
There are two files in this pr
Remove one

File: share/goodie/cheat_sheets/json/libre-office-impress.json [hunk 5]
```diff
@@ -0,0 +1,286 @@
+{
+	"id": "libre_office_impress_cheat_sheet",
+	"name": "LibreOffice Impress",
+	"description": "Keyboard shortcut for LibreOffice Impress",
+	"metadata": {
+             "sourceName": "LibreOffice Help",
+             "sourceUrl":  "https://help.libreoffice.org/Impress/Shortcut_Keys_for_Impress"
+	},
+	"template_type": "ksdeyboard",
```
Comment (Tawsif1) [inline share/goodie/cheat_sheets/json/libre-office-impress.json]: Change template type to "keyboard"

**Comment by pjhampton** [comment]
This PR looks to have gone stale. Closing 👍 

If you are looking for something more substantial to work through that will have a big impact for the users of DuckDuckGo, please check out the [DuckDuckHack Issues Table](https://duckduckhack.com/issues). 🚀  😄

**Pull request closed without merging**